### ISSUE TITLE
No Audio When Using Dwarf-to-Kobold Voice Pack

### ISSUE THREAD

**Issue by KyrieTheBat** [trumank/mint#114]
When using this mod: https://mod.io/g/drg/m/dwarf-to-kobold-voice-pack#1966637

Several bits of audio are missing, namely the new voice lines. Other bits of audio I immediately tell are missing are the space rig bunk doors are silent, as well as player footsteps.

**Comment by trumank** [comment]
And that is the only mod you have installed? I installed it and the door open/close sounds are working fine for me, as well as footsteps.

**Comment by KyrieTheBat** [comment]
Yeah, it’s the only mod I have enabled in Mint. I went through modio and disabled/unsubscribed from everything just in case. In your case do the voice lines work too?

**Comment by QuentinMcCarthy** [comment]
I also have this issue, except I have the reverse. All audio becomes excessively loud, and there's overlaps with voice lines.
